alexa_media/media_player.py

```diff
@@ -520,7 +518,9 @@ def set_volume_level(self, volume):
             return
         self.alexa_api.set_volume(volume)
         self._media_vol_level = volume
-        self.update()
+        if not (self.hass.data[DATA_ALEXAMEDIA]
```

Comment:
Same for this, assumption is websockets will always be active, let's make sure to remove these lines of code in the future and rely solely on event messages.